snuba/request/request_settings.py

```diff
@@ -34,3 +34,20 @@ def get_rate_limit_params(self) -> Sequence[RateLimitParameters]:
 
     def add_rate_limit(self, rate_limit_param: RateLimitParameters) -> None:
         self.__rate_limit_params.append(rate_limit_param)
+
+
+class SubscriptionRequestSettings(RequestSettings):
+    def __init__(self, turbo: bool, consistent: bool, debug: bool) -> None:
+        self.__debug = debug
```

Comment:
This won't work because of name mangling, it'd actually end up unbound because `super` is never called.

(mypy doesn't catch this because it doesn't implement name mangling, and it's the only real glaring issue I've run into with it.)

Reply:
Damn, good point.